### ISSUE TITLE
Dialog rails embeddings only similarity threshold 

### ISSUE THREAD

**Issue by Serbernari** [NVIDIA/NeMo-Guardrails#345]
Hi! I would like to use embeddings_only dialog rails which will fall into fallback:

``` 
define flow ... 
    priority 0.1
    user ...
    bot responds truthfully
````

when semantic similarity between canonical user forms and given query is smaller than, lets say, 0.75. How could I implement that? Is this parameter available in config?

**Comment by drazvan** [comment]
@Serbernari : currently we don't have support for a similarity threshold in `embeddings_only` mode. I'll mark this as a feature request. If you're interested in contributing this, let me know and I can guide you.

**Comment by ZicsX** [comment]
Hi @drazvan, I am also very interested in having a similarity threshold feature in the embeddings_only dialog rails mode, with a fallback mechanism for cases where the similarity score falls below a specified threshold (e.g., 0.75). This would greatly enhance the robustness of our system. Could you please provide guidance or an outline of how to implement this feature?

**Comment by Pouyanpi** [comment]
This feature is implemented and available in v0.10.0.
